Java/libraries/recognizers-text-number/src/main/java/com/microsoft/recognizers/text/number/parsers/BaseCJKNumberParser.java

```diff
@@ -172,7 +172,7 @@ protected ParseResult parsePercentage(ExtractResult extResult) {
 
                     if (intNumberChar == '対' || intNumberChar == '对') {
```

Comment:
I know it's slightly out of scope for this exact PR, but can you move this "FiveChars" also to YAML resources?